docs/website/content/v0.3/en/configuration/v1alpha1.md

```diff
@@ -0,0 +1,276 @@
+```
+
+nolint: lll
+
+## Reference
+
+### type
+
+Type: `string`
+
+Valid Values:
+
+- ``init``
+- ``controlplane``
+- ``join``
+
+The machine's `type` defines the role of the machine within the cluster.
+
+#### Init
+
+Init node type designates the first control plane node to come up.
+You can think of it like a bootstrap node.
+This node will perform the initial steps to bootstrap the cluster -- generation of TLS assets, starting of the control plane, etc.
+
+#### Control Plane
+
+Control Plane node type designates the node as a control plane member.
+This means it will host etcd along with the Kubernetes master components such as API Server, Controller Manager, Scheduler.
+
+#### Worker
+
+Worker node type designates the node as a worker node.
+This means it will be an available compute node for scheduling workloads.
+
+### token
+
+Type: `string`
+
+The `token` is used by a machine to join the PKI of the cluster.
+Using this token, a machine will create a certificate signing request (CSR), and request a certificate that will be used as its' identity.
+
+Examples:
+
+```yaml
+token: 328hom.uqjzh6jnn2eie9oi
+```
+
+> Warning: It is important to ensure that this token is correct sinc a machine's certificate has a short TTL by default
```

Comment:
sp  s/sinc/since/

Reply:
think this warning could use additional context / detail. 
since the token gets generated, id assume most people politely ignore it. so there should be low chance of the token being incorrect. but, in the case that the token does get truncated during editing, what does that actually translate to? i don't think the certificate ttl comes in to play at this point because most likely the <broken> machine won't be able to talk to trustd to request a certificate.

Reply:
It comes into play if/when someone wants to or accidentally changes the token, say for rotation. Its important to get makes sure the nodes have the correct token because they renew so often. Since we don't have a way, yet, to update a machine's token, this means bouncing the node, but still important to point out that they could potentially lose node access fairly quickly if they don't update worker tokens after updating master tokens.